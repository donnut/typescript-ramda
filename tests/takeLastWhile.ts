import R_takeLastWhile = require('../ramda/dist/src/takeLastWhile');

declare const string_to_boolean: (x: string) => boolean;
declare const string_array: string[];

// @dts-jest:pass:snap
R_takeLastWhile(string_to_boolean);
// @dts-jest:pass:snap
R_takeLastWhile(string_to_boolean, string_array);
