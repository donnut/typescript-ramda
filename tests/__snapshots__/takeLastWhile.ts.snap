// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R.takeLastWhile(string_to_boolean) 1`] = `"(list: string[] | ArrayLike<string>) => string[]"`;

exports[`R.takeLastWhile(string_to_boolean, string_array) 1`] = `"string[]"`;
